apps/gkyl_gyrokinetic_multib_priv.h

```diff
@@ -58,6 +58,29 @@ struct gyrokinetic_multib_output_meta {
   const char *topo_file_name; // name of topology file
 };
 
+
+// field data
+struct gk_field_multib {
+  struct gkyl_gyrokinetic_multib_field info; // data for field
+  enum gkyl_gkfield_id gkfield_id; // type of field
+
+
+  struct gkyl_multib_comm_conn **mbcc_send; // comm object for allgather
+  struct gkyl_multib_comm_conn **mbcc_recv; // comm object for allgather
+  struct gkyl_range **multibz_ranges; // ranges for allgather along z
+  struct gkyl_range **multibz_ranges_ext; // extended ranges for smoothing along z
+  struct gkyl_range **block_subranges; // ranges for copying smooth charge density
```

Comment:
same thing here, specify this is along z, maybe call it `block_subrangesz` ?
In general keep in mind that we soon want to parallelize in x-y too, so we'll have to do some gathers/copies in other directions too. By naming things with specificity you make that other task easier